cli/tests/data/json-body-required.txt

```diff
@@ -8,6 +8,7 @@ oxide silo policy update
 oxide vpc firewall-rules update
 oxide vpc router route create
 oxide vpc router route update
+oxide alert receiver webhook create
```

Comment:
this is due to `secrets: Vec<String>` being required in the body. We can/should update progenitor CLI to turn this into an arg that may be multiply specified e.g. `--secret x --secret y`